### ISSUE TITLE
`PartitionGetInfo error` prevents booting (even to fastboot or ECL)

### ISSUE THREAD

**Issue by sarayourfriend** [ProjectValhalla/OdinMultiBootGuides#23]
I experienced a power outage while trying to flash my Odin Pro (M1) back to just android using the restore script that cut the process short. The unit will now no longer boot to anything and I can't get to fastboot to reflash it and it seems that I'm not able to get it into ECL mode either to use QFIL.

This is the full error:

```
[kevin]Partition qpdata1 not found error
[kevin]GetPartitionInfo error
```

Here's what I've tried so far:

1. Regular fastboot, volume down + power. It powers on, flashes the Odin logo, and then displays the error above.
2. ECL mode, volume down + volume up (+ power, I've tried with and without, following different disparate advice around). It does the same as trying to get into fastboot.
3. Jumping some test points. I found a video in the Ayn Odin Facebook group that supposedly allows you to recover the device (somehow) from absolute disaster situations like this one. I jumped the two test points in the video (the top two in the cluster where the bottom of the group aligns horizontally with the battery connector) but I honestly have no idea if I've done it correctly because the video is quite vague about when exactly you're meant to jump them. After doing this several times and trying various different tricks (jumping them while plugging it in, etc) I haven't been able to get the behaviour to change and honestly I'm not sure what to expect anyway.

Do y'all have any advice for further troubleshooting steps I should try or any ideas for how to get out of this semi-bricked state? 

Thanks for the exciting project. It was great to fiddle around with Debian while I had it installed.

**Comment by dfilskov** [comment]
That doesn't sound good.

Perhaps you could ask AYN's representatives in the support channel on their official Discord server.

**Comment by sarayourfriend** [comment]
Do you have a link to the official Discord? I've asked in the Taki Udon discord but no one seems to have any ideas (no responses at all), and I've emailed Ayn to no avail.

**Comment by dfilskov** [comment]
Sure - it's here it is:

https://www.ayntec.com/pages/contact-us

**Comment by dfilskov** [comment]
Do you need to Discord server for this Valhalla project too?

**Comment by sarayourfriend** [comment]
The Ayn Discord linked me to it, thanks. I'll update here once I have a resolution (or if there isn't one within the next month, I'll also leave an update saying so).

**Comment by sarayourfriend** [comment]
I finally got a solution back from Ayn that worked perfectly, with some caveats that I think were due to low battery life (the device had been sitting untouched since this issue happened).

To get the device back into a flashable state: **Plug the device into the computer, then hold the `power`, `volume +` and `volume -` all together for (at least) 10 seconds**. This didn't work for me at first. It would flash a low battery symbol with a plug inside the battery, then go straight to the boot error described in this issue. At first the red charging LED was flashing red, then it turned solid red, then finally turned off. For each of these states, I was still unable to get the device to be recognised by QFIL (or the Windows 11 PC at all). It was only once the Odin logo showed on the screen _alongside the boot error_ that I was able to hold all three buttons for 10 seconds and the computer recognised it. It immediately showed in QFIL and I was able to flash the device no problem. The Odin is currently sitting on the setup screen.

Hopefully this helps anyone else who gets into this or a similar pickle. The lesson was: **Plug it into the computer and hold all three buttons for 10 seconds**. That gets it into a flashable state for QFIL.
